test/parallel/test-throw-default-error-unhandled-rejections.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const spawn = require('child_process').spawn;
+const node = process.execPath;
+
+if (process.argv[2] === 'child') {
+  const rejectPromise = () => {
+    return new Promise((resolve, reject) => {
+      setTimeout(reject(new Error('Reject Promise')), 100);
```

Comment:
If you want to have the `reject()` call delayed, you need to wrap it in a closure, i.e. `setTimeout(() => reject(new Error('Reject Promise')), 100);`. Right now, the whole `child` block is equivalent to `Promise.reject(new Error('Reject Promise'))`.